internal/host/validator.go

```diff
@@ -740,19 +740,15 @@ func (v *validator) hasSufficientPacketLossRequirementForRole(c *validationConte
 	if len(c.host.Connectivity) == 0 {
 		return ValidationPending
 	}
-	hosts, err := v.validatePacketLossForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
-	if err != nil || len(hosts) > 0 {
-		return ValidationFailure
-	}
-	return ValidationSuccess
+	status, _, _ := v.validatePacketLossForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
+	return status
 }
 
-func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host) ([]string, error) {
-
-	var connectivityReport *models.ConnectivityReport
+func (v *validator) validatePacketLossForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host) (ValidationStatus, []string, error) {
 	connectivityReport, err := hostutil.UnmarshalConnectivityReport(host.Connectivity)
 	if err != nil {
-		return nil, fmt.Errorf("unable to unmarshall host connectivity for %s:%s", host.ID, err)
+		v.log.Warnf("unable to unmarshall host connectivity for %s:%s", host.ID, err)
+		return ValidationFailure, nil, err
```

Comment:
Why remove the formatted error from return ?

Reply:
I removed it because It was duplicating the text in the error for the unmarshalling ("unable to unmarshall... : json: cannot unmarshal... ". Leaving it as part of the log made more sense than expose this repetitive message. Do you want me add it back?

Reply:
So why do you need it in the log message ?

Reply:
Because I wanted as much detail as possible in the log message, but I didn't want to expose that detail to the user.